tensorboard/webapp/metrics/views/card_renderer/scalar_card_types.ts

```diff
@@ -72,10 +72,19 @@ export interface PartitionedSeries {
 }
 
 /**
- * This enum defines the columns available in the data table. The
- * ScalarCardComponent must know which piece of data is associated with each
- * value and the DataTable widget must know how to display each value.
+ * An object which is intended to hold the min and max step within each scalar
+ * card.
  */
+export type MinMaxStep = {
+  minStep: number;
+  maxStep: number;
+};
+
+/**
+ * THE SECTION BELOW IS HERE TO AVOID BREAKING SYNC. THESE SHOULD NOT BE USED!!!
+ * Please use webapp/widgets/data_table/types for these types.
+ */
```

Comment:
Because I know this is super temporary it's fine but I would rather you import these types from data_table_types then re export them

Reply:
I went ahead and did that.